README.md

```diff
@@ -1,24 +1,107 @@
 # BrightSign CLI API Tool
+<p align="center">
+    <img src="./readmeMedia/PoweredByPurple.jpg" alt="Powered by Purple!" width="400" height="194">
+</p>
 The purpose of this package is to allow users to communicate with the BrightSign's Local DWS (Diagnostic Web Server) REST HTTP APIs through a simple CLI tool, `bsc`. 
 
```

Comment:
maybe make this the link e.g. [Local DWS (Diagnostic Web Server) REST HTTP APIs](https://brightsign.atlassian.net/wiki/spaces/DOC/pages/1172734089/Local+DWS+APIs)

Reply:
Updated - and updated the image to link to the same site for now.